src/webrtc/call.ts

```diff
@@ -394,6 +396,7 @@ export class MatrixCall extends EventEmitter {
 
             logger.debug("Got screen stream, requesting audio stream...");
             const audioConstraints = getUserMediaContraints(ConstraintsType.Audio);
+            this.type = CallType.Video;
```

Comment:
Is there any reason to put this inside and not before the try-catch block? I am probably just missing something here :sweat_smile:

Reply:
Oh no , sorry I would just fix it

Reply:
Hmm, I've just realized this might actually do something we want by accident :D

Reply:
Oh, just a sec let me just check . I think it would work the either way

Reply:
I think an intelligent thing to do here would be to decouple the call type and the `NoUserMedia` error. But I wouldn't worry about it if you don't want to

Reply:
The current solution will work fine

Reply:
I could do that but I've a naive question how do you make screen share calls here? I couldn't figure out

Reply:
Screen-share calls are a bit of a hack right now... You can shift-click the video call button to initiate a screen-sharing call

Reply:
> The current solution will work fine

Alright